Makefile

```diff
@@ -90,10 +90,7 @@ install-shell-completion:
 	sudo cp scripts/contrib/completion/bash/netctl /etc/bash_completion.d/netctl
 endif
 
-build:
-	make start
-	make ssh-build
-	make stop
+build: start ssh-build stop
```

Comment:
We're already using something similar in the following places:
https://github.com/contiv/netplugin/blob/master/Makefile#L33
https://github.com/contiv/netplugin/blob/master/Makefile#L81
https://github.com/contiv/netplugin/blob/master/Makefile#L188
https://github.com/contiv/netplugin/blob/master/Makefile#L227

We would've run into this already. Please let me know if you run into any issues, with or without this PR.